dashboards/qa-metrics.erb

```diff
@@ -10,7 +10,26 @@
   </li>
 
   <li data-row="2" data-col="1" data-max-sizey="1" data-sizex="1" data-sizey="1">
-    <div data-view="Countdown" data-title="Next FEATURE FREEZE" data-end="14-August-2020 17:00:00"></div>
+    <%
+      require 'date'
+
+      deadline = nil
+      the_date = Date.today
+      #puts "the_date = #{the_date}"
+
+      if the_date < (the_date.next_month - the_date.next_month.mday + 1)
```

Comment:
Does this need change to support the new cadence of feature freeze on the 2nd Friday and code freeze on the 4th Friday of the month